lib/jxl/encode_test.cc

```diff
@@ -367,7 +367,10 @@ TEST(EncodeTest, OptionsTest) {
               JxlEncoderOptionsSetInteger(
                   options, JXL_ENC_OPTION_MODULAR_PREDICTOR, 14));
     VerifyFrameEncoding(enc.get(), options);
-    EXPECT_EQ(30, enc->last_used_cparams.colorspace);
+    // It was set to 30, but becomes 32 because in the C++ implementation, the
+    // numerical RCT values are shifted 2 compared to the specification. The
+    // API uses the values seen in the JXL specification.
+    EXPECT_EQ(32, enc->last_used_cparams.colorspace);
```

Comment:
It might make sense to also make the C++ implementation just follow the spec. The weird numbering is a historical thing from a time when YCoCg was still a separate modular transform and RCT didn't exist yet.

Reply:
I'd do that in a separate MR, the goal here is to add all cjxl options as options to the API so that cjxl can use it. The internal C++ implementation is separate and mixing changing behavior with the API makes it more difficult to revert MR's and find out which one causes an issue.